implementations/elixir/ockam/ockam_metrics/test/ockam_metrics/telemetry_poller_test.exs

```diff
@@ -0,0 +1,111 @@
+defmodule Ockam.Metrics.TelemetryPoller.Tests do
+  use ExUnit.Case, async: true
+  doctest Ockam.Metrics.TelemetryPoller
+
+  alias Ockam.Identity
+  alias Ockam.Metrics.TelemetryPoller
+  alias Ockam.SecureChannel
+  alias Ockam.Vault.Software, as: SoftwareVault
+
+  setup do
+    Ockam.Node.register_address("test")
+    on_exit(fn -> Ockam.Node.unregister_address("test") end)
```

Comment:
If we use a fixed node name that can collide, should this module be `async: false`?

Reply:
good question :thinking: